[2021-09-27 14:07:18,358] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_complex.create_entry_group_result2 2021-09-27T14:07:06.295229+00:00 [queued]>
[2021-09-27 14:07:18,373] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_complex.create_entry_group_result2 2021-09-27T14:07:06.295229+00:00 [queued]>
[2021-09-27 14:07:18,374] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 14:07:18,376] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-27 14:07:18,377] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 14:07:18,395] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): create_entry_group_result2> on 2021-09-27T14:07:06.295229+00:00
[2021-09-27 14:07:18,400] {standard_task_runner.py:52} INFO - Started process 2157 to run task
[2021-09-27 14:07:18,405] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'example_complex', 'create_entry_group_result2', '2021-09-27T14:07:06.295229+00:00', '--job-id', '6', '--pool', 'default_pool', '--raw', '--subdir', '/home/***/.local/lib/python3.6/site-packages/***/example_dags/example_complex.py', '--cfg-path', '/tmp/tmpfkz0ydli', '--error-file', '/tmp/tmppwtyl4m0']
[2021-09-27 14:07:18,408] {standard_task_runner.py:77} INFO - Job 6: Subtask create_entry_group_result2
[2021-09-27 14:07:18,476] {logging_mixin.py:109} INFO - Running <TaskInstance: example_complex.create_entry_group_result2 2021-09-27T14:07:06.295229+00:00 [running]> on host 6c9f6f41e1ed
[2021-09-27 14:07:18,544] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_complex
AIRFLOW_CTX_TASK_ID=create_entry_group_result2
AIRFLOW_CTX_EXECUTION_DATE=2021-09-27T14:07:06.295229+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-09-27T14:07:06.295229+00:00
[2021-09-27 14:07:18,548] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-09-27 14:07:18,553] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo create_entry_group_result2']
[2021-09-27 14:07:18,564] {subprocess.py:74} INFO - Output:
[2021-09-27 14:07:18,566] {subprocess.py:78} INFO - create_entry_group_result2
[2021-09-27 14:07:18,568] {subprocess.py:82} INFO - Command exited with return code 0
[2021-09-27 14:07:18,612] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=create_entry_group_result2, execution_date=20210927T140706, start_date=20210927T140718, end_date=20210927T140718
[2021-09-27 14:07:18,663] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-27 14:07:18,693] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
